src/components/Dashboard/cells/sponsors.js

```diff
@@ -0,0 +1,132 @@
+import jackson from '../../../assets/sponsors/jackson.png';
+import mirus_research from '../../../assets/sponsors/mirus_research.png';
+import mozilla from '../../../assets/sponsors/mozilla.png';
+import npm from '../../../assets/sponsors/npm.svg';
+import nvisia from '../../../assets/sponsors/nvisia.png';
+import optum from '../../../assets/sponsors/optum.png';
+import particle from '../../../assets/sponsors/particle.png';
+import rc from '../../../assets/sponsors/rc.png';
+import schlum from '../../../assets/sponsors/schlum.png';
+import snl from '../../../assets/sponsors/snl.png';
+import two_sigma from '../../../assets/sponsors/two_sigma.png';
+import zeit from '../../../assets/sponsors/zeit.svg';
+
+//https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array/2450976#2450976
+function shuffle(array) {
+  var currentIndex = array.length, temporaryValue, randomIndex;
+
+  // While there remain elements to shuffle...
+  while (0 !== currentIndex) {
+
+    // Pick a remaining element...
+    randomIndex = Math.floor(Math.random() * currentIndex);
+    currentIndex -= 1;
+
+    // And swap it with the current element.
+    temporaryValue = array[currentIndex];
+    array[currentIndex] = array[randomIndex];
+    array[randomIndex] = temporaryValue;
+  }
+
+  return array;
+}
+
+class Lane extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = this.initializeState();
+    
+    this.initializeState = this.initializeState.bind(this);
+  }
+
+  initializeState() {
+  }
+
+  render(){
+    return (
+      <div className = "lane">
+        <div className = "car">
+          <img src={this.props.myimage} alt = "sponsor-logo"/>
+        </div>
+      </div>
+    )
+  }
+}
+
+class Sponsors extends React.Component{
+  constructor(props){
+    super(props);
+    //making this a member variable so that I can close the interval when closing
+    this.interval = null;
+
+    //list of all our sponsors
+    this.images = [amadeus, bp, c1, caterpillar, citadel, facebook, fulcrum, google,
+    imc, jackson, mirus_research, mozilla, npm, nvisia, optum, particle,
+    rc, schlum, snl, two_sigma, zeit];
+
+    //will contain a list of randomly generated indices
+    var array = new Array(this.images.length);
+    for (var i=0; i<array.length; i++) {
+      array[i] = i;
+    }
+    array = shuffle(array);
+    
+    this.state = {
+      indices: array,
+      offset: 0,
+      refreshrate: 8 * 1000 //ensure this integer matches the animation duration
+    }
+
+    this.getImage = this.getImage.bind(this);
+    this.updateImages = this.updateImages.bind(this);
+  }
+
+  updateImages(){
+    var newOffset = (this.state.offset+4) % this.state.indices.length;
+    this.setState({
+      offset: newOffset
+    });
+  }
+
+  componentDidMount(){
+    this.interval = setInterval(this.updateImages, this.state.refreshrate);
+  }
+
+  componentWillUnmount(){
+    clearInterval(this.interval);
+  }
+
+
```

Comment:
Get rid of this extra space. Or you could wait until the linter PR goes through, then you can just run the linter on this